Apps/Contoso.UWP.Puppet/MainPage.xaml.cs

```diff
@@ -23,6 +24,11 @@ public MainPage()
         {
             this.InitializeComponent();
             Application.Current.UnhandledException += OnUnhandledException;
+            object storageSize;
```

Comment:
Same

Reply:
There is no _Settings.Default_ for UWP.

Reply:
Can we use constant instead?

Reply:
These settings contains only objects. That is why the code was written in a such way.